code/modules/virus2/effect.dm

```diff
@@ -377,6 +377,37 @@
 	speech.message = message
 
 
+/datum/disease2/effect/vitreous
+	name = "Vitreous resonance"
+	stage = 2
+	chance = 25
+	max_chance = 75	
+	max_multiplier = 2
+
+datum/disease2/effect/vitreous/activate(var/mob/living/carbon/mob)
```

Comment:
You can compress the istype and cast below by making this variable casted to humans, then simply making the istype have a single argument so it goes by variable name.

Then you don't need to cast on a separate line.